routes/index.js

```diff
@@ -38,22 +45,121 @@ router.get('/login', function (req, res, next) {
 //get breakfast menu page
 router.get('/breakfast', function (req, res, next) {
     Breakfast.find(function (err, docs) {
-        res.render('breakfast', {title: 'Breakfast Menu', breakfast: docs});
+        res.render('menus/breakfast', {title: 'Breakfast Menu', breakfast: docs});
+    });
+});
+
+router.get('/add-to-cart/breakfast/:id', function (req, res, next) {
+    let bfItemId = req.params.id;
+    let cart = new Cart(req.session.cart ? req.session.cart : {});
+    Breakfast.findById(bfItemId, function (err, bfItem) {
+        if(err){
+            return res.redirect('/', {title: 'Something went wrong.'});
+        }
+        cart.add(bfItem, bfItem.id);
+        req.session.cart = cart;
+        console.log(req.session.cart);
+        res.redirect('/breakfast');
     });
 });
 
 //get lunch page
 router.get('/lunch', function (req, res, next) {
     Lunch.find(function (err, docs) {
-        res.render('lunch', {title:'Lunch Menu', lunch: docs});
+        res.render('menus/lunch', {title:'Lunch Menu', lunch: docs});
+    });
+});
+
+router.get('/add-to-cart/lunch/:id', function (req, res, next) {
+    let lunchItemId = req.params.id;
+    let cart = new Cart(req.session.cart ? req.session.cart : {});
+    Lunch.findById(lunchItemId, function (err, lunchItem) {
+        if(err){
+            return res.redirect('/', {title: 'Something went wrong.'});
+        }
+        cart.add(lunchItem, lunchItem.id);
+        req.session.cart = cart;
+        console.log(req.session.cart);
```

Comment:
ℹ️ Careful with all of your console.log. This is going to become hard to debug if theres a significant amount of unwanted logs happening.

Reply:
@reactivepixel fixed it https://github.com/piazzaana/chicken-fries/pull/66